### DISCUSSION TITLE
Seller name, order history, search page, seller distance, wfcm disable multivendor checkout

### DISCUSSION THREAD

**Discussion by exploitedtaken** [inspireui/support#5160] [New Feature/Ideas Requests]
Hello, there is still no seller name in the order history and product search section. When products are purchased from 2 different vendors, it is not known which product belongs to which seller on the order history page. The seller still has no distance to the user. Aren't these basically supposed features? When you register via SMS, the bug still continues. It does not give a warning when a product is requested to be purchased from more than one seller in the same order. Complete your order at a vendor first, then it should give a warning like create a new order. I use this feature on the site in the WCFM plugin. Since this feature is not available in the application, I cannot use it efficiently. When purchasing products from 2 different sellers on the application, it does not appear which product belongs to which seller. At the same time, when we want to cancel a product in the order, the order is completely canceled. I have been waiting for this update for 2 months. You conveyed what I said to the team that you will add them in the next update.

**Comment by exploitedtaken** [comment]
Hello, there is a problem, let's say, when the products were purchased from 2 different vendors, the FluxStore Manager application was logged into the account of the dealer from whom the product was purchased. With both sellers' orders visible, only that seller's order should appear. When the order status is updated from the logged-in seller account, the order information for both sellers is updated. However, if the following feature "disable multivendor checkout" in wcfm is added to the application, this situation will be automatically resolved.

**Reply by CeoAlexMartinez** [reply]
Esperando estas solucion

**Comment by exploitedtaken** [comment]
No seller name in the order history and product search section. seller distance, wfcm disable multivendor checkout 

Not in new update :(

**Comment by exploitedtaken** [comment]
I hope these features are included in the new update. At least, the distance of the seller to the user and order tracking and the name of the seller should be in the product search section. Now, when my customers buy products from more than one seller in an order, they constantly ask which product was bought from which seller.

**Comment by exploitedtaken** [comment]
Hi, when will the new update come? Will the seller name in the order tracking section, the seller name in the product search section, and the distance of the seller from the user to be added in this update? When users place an order, they do not know from which seller they bought the product and they constantly ask question. You also know that when products are received from more than one seller in an order, when one seller completes the order, the order is completed for both sellers, even though the order is placed from two sellers.

Just for this reason, the "disable multi vendor checkout" option in wcfm must be added to the application.

I am waiting your good news :)

**Comment by exploitedtaken** [comment]
I am still waiting :( not in this update

**Comment by exploitedtaken** [comment]
Yes, in the latest version of the application, the feature of showing the distance to the user of the business has now arrived. Thank you guys.  But still the name of the seller is not written in the order history section.

While searching for a product, the name of the seller still does not appear without clicking on the product.

Since "Wfcm disable multivendor checkout" has not been added yet, when a product is received from more than one store, when a seller says the order is complete in the system, the order of two different stores is completed because the other seller's product is also included in the same order number.

**Comment by CeoAlexMartinez** [comment]
esperando actualizacion

**Comment by exploitedtaken** [comment]
Hi, after the last update of MStore API, before only 10% of the categories were visible before, now they all gone. I had to downgrade to Version 2.3.0 which was released on July 15th, where all the categories were visible and the vendors were showing properly on the map.

Version 2.3.0 everything works correctly in this version. Please check. But 2.4.0 + 2.4.1 is not working correctly. Including the issues I mentioned earlier below.

You guys added nice features, but I wish these problems were not there, then I can buy an extended license.

Problems I had with these versions 2.4.0 + 2.4.1

* The categories are not loading. When I choose dokan or woo the env file’s type, all the missing categories are loaded. But when I make the type wcfm, most of the categories still don’t load.

* I actively use the variation option in WCFM. eg. 1st burger in variation product. 2. Burger and drink options are active. There is no problem when I choose the options and add them to the cart. However, when I come to the payment section, I get a please select variations warning.

* I am using Woocommerce Product Add-Ons plugin.

For example, on the application. Burger $34.90 + Fries and Drink $9.95 when added to the cart, the price is reflected in the total. However, when it comes to payment, the price of the extra option is not reflected. Only the $ 34.90 price of the product appears.
